lib/book_settings.py

```diff
@@ -144,12 +143,7 @@ def search_for_asin(self, query):
         except Exception as e:
             try:
                 self._aConnection.close()
-                if self._proxy:
-                    self._aConnection = HTTPSConnection(self._https_address, self._https_port)
-                    self._aConnection.set_tunnel('www.amazon.com', 443)
-                else:
-                    self._aConnection = HTTPSConnection('www.amazon.com')
-
+                self._aConnection.connect()
```

Comment:
OK - odd that it helps doing it manually (vs. the auto-close) but if it has a benefit then keep it.  Smaller/simpler now anyway.